addons/l10n_it_edi/models/account_move.py

```diff
@@ -1082,18 +1078,23 @@ def _l10n_it_edi_get_extra_info(self, company, document_type, body_tree, incomin
             'type_tax_use_domain': [('type_tax_use', '=', 'purchase' if incoming else 'sale')],
         }, []
 
-    def _l10n_it_edi_import_invoice(self, invoice, data, is_new):
-        """ Decodes a l10n_it_edi move into an Odoo move.
+    def _l10n_it_edi_import_invoice(self, file_data):
```

Comment:
Be aware of possible subclasses. I just `r+`ed this.
https://github.com/odoo/odoo/pull/200774/files#diff-3099156af572bf8a1c157507ba1b1f6bcdc0beaa434591c3e18a8dff615401b0R243